NMS-11747: cleanup monitor docs common parameters (#4407)
OpenNMS/opennms: docs/modules/reference/pages/service-assurance/introduction.adoc

| invert-status
| Invert the up/down behavior of the monitor.
| false

| ds-name *
| Name of the RRD data source (DS) name in which to store this service monitor's response-time samples.
| response-time

| rrd-base-name *

> | The file name of the RRD file (minus the `.rrd` or `.jrb` file extension) within the specified rrd-repository path in which this service monitor's response-time samples will be persisted.

Reviewer: ```suggestion
| The file name of the RRD file (minus the `.rrd` or `.jrb` file extension) within the specified rrd-repository path in which to persist this service monitor's response-time samples.
```
